packages/kolibri-common/composables/useTaskPooling.js

```diff
@@ -0,0 +1,48 @@
+import logger from 'kolibri-logging';
+import { ref, onMounted, onUnmounted } from 'vue';
+import { useTimeoutPoll } from '@vueuse/core';
+import TaskResource from 'kolibri/apiResources/TaskResource';
+
+const taskPollers = new Map();
+
+const logging = logger.getLogger(__filename);
+
+export default function useTaskPooling(queueName) {
```

Comment:
I assume this was a typo, but could we call this `useTaskPolling` rather than `useTaskPooling`? I guess it is kind of doing the latter, by pooling our polling of tasks, but the polling is what we really care about!